app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -46,110 +47,64 @@ class Settings private constructor(
 
     private val appContext = context.applicationContext
 
-    val preferences: SharedPreferences =
+    override val preferences: SharedPreferences =
         appContext.getSharedPreferences(FENIX_PREFERENCES, MODE_PRIVATE)
 
-    val usePrivateMode: Boolean
-        get() = preferences.getBoolean(appContext.getPreferenceKey(R.string.pref_key_private_mode), false)
+    var usePrivateMode by booleanPreference(appContext.getPreferenceKey(R.string.pref_key_private_mode))
```

Comment:
nit: maybe just a preference of mine, but I feel like it's better to be explicit about the default state of these preferences. Is there a reason you prefer not just passing in a `default` state always? It makes reading through this file a lot easier IMO.